std/experimental/checkedint.d

```diff
@@ -1641,13 +1641,24 @@ static:
 
 @system unittest
 {
+    // Temporarily disable output to stderr so as to not spam the build log.
+    import std.stdio;
+    auto t = stderr;
+    stderr.open("/dev/null", "w");
```

Comment:
This won't work on Windows.

Reply:
The name on windows is `nul`, correct?

Reply:
I don't know much about Windows, sorry.
However, it might be better to do a `version (StdUnittest)` at the beginning to define a different stderr temp file as this will avoid all the shenanigans with rewriting the global output. However, it does come with its own set of problems...

Reply:
I thought it was `NUL:`, but wikipedia seems to agree with you: https://en.wikipedia.org/wiki/Null_device

Reply:
I think the most robust solution is to write to a temp file and make sure it contains what it's supposed to. Will be back.